src/Components/v2/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx

```diff
@@ -1,31 +1,60 @@
-import { Flex, PriceRange, Toggle } from "@artsy/palette"
-import React, { FC } from "react"
+import React from "react"
+import { Flex, Radio, RadioGroup, Toggle } from "@artsy/palette"
 import { useArtworkFilterContext } from "../ArtworkFilterContext"
-import { MAX_PRICE, MIN_PRICE } from "../Utils/rangeToTuple"
 
-export const PriceRangeFilter: FC = () => {
+export const PriceRangeFilter: React.FC = () => {
   const filterContext = useArtworkFilterContext()
-  const [initialMin, initialMax] = filterContext.rangeToTuple("priceRange")
+  const initialRange = filterContext.filters.priceRange
 
   return (
     <Toggle label="Price" expanded>
       <Flex flexDirection="column" alignItems="left" my={1}>
-        <PriceRange
-          allowCross={false}
-          min={MIN_PRICE}
-          max={MAX_PRICE}
-          step={50}
-          disabledText="Disabled for biddable works"
-          defaultValue={[initialMin, initialMax]}
-          disabled={Boolean(filterContext.filters.atAuction)}
-          onAfterChange={([min, max]) => {
-            const minStr = min === MIN_PRICE ? "*" : min
-            const maxStr = max === MAX_PRICE ? "*" : max
-
-            filterContext.setFilter("priceRange", `${minStr}-${maxStr}`)
+        <RadioGroup
```

Comment:
This looks pretty straightforward!

Would you mind attaching a screenshot or gif to the description? 🙏 

Also I don't know if there are tests for this, but it would be awesome to have a test that verifies things like the disabled state + selecting an option sends the correct param to the API.

Reply:
Done. I tried a gif but it's liek 100MB so it doesn't work. 😅

I'm playing around with the test. There is the disabled test [here](https://github.com/artsy/palette/blob/b21d6b147b9d2b0fcb5b8a4849fddca1f295a0f1/packages/palette/src/elements/RadioGroup/RadioGroup.test.tsx#L102) but I'll try the param one.